controllers/users/currentUser.js

```diff
@@ -0,0 +1,16 @@
+import { getUserById } from "../../service/index.js";
+
+async function currentUser(req, res, next) {
+  if (res.user.length === 0) {
+    return res.status(401).json({ message: "Not authorized" });
```

Comment:
To powinna być w middleware jwt a nie tu. Czyli tu osoba nie powinna się znaleźć jeśli ma token nie prawdziwy albo jeśli user nie istnieje.